ec2Bootstrap/terminateSelf.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/bash
+
+# Script to terminate instance, spot instance will automatically be recreated later
```

Comment:
The logs were problematic before yeah, also had issues where client would not be correctly shutdown. Made some changes to prevent that but would like to test first before.